client/components/jetpack-product-card/features-item.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import InfoPopover from 'components/info-popover';
+import { preventWidows } from 'lib/formatting';
+import { FeaturesItem } from './types';
```

Comment:
We can use `import type` instead of `import` here.
https://www.typescriptlang.org/docs/handbook/modules.html#importing-types

Reply:
Didn't know about that. Thanks @rcanepa!